package.json

```diff
@@ -46,36 +46,36 @@
     "@angular/platform-browser": "9.1.12",
     "@angular/platform-browser-dynamic": "9.1.12",
     "@angular/router": "9.1.12",
-    "@blackbaud/auth-client": "2.35.0",
+    "@blackbaud/auth-client": "2.37.0",
     "@blackbaud/skyux-lib-clipboard": "4.0.0",
     "@blackbaud/skyux-lib-code-block": "4.0.1",
     "@blackbaud/skyux-lib-media": "4.0.0",
     "@blackbaud/skyux-lib-restricted-view": "4.1.0",
     "@blackbaud/skyux-lib-stache": "4.2.1",
-    "@skyux-sdk/builder": "4.5.0",
-    "@skyux-sdk/builder-plugin-skyux": "4.1.3",
+    "@skyux-sdk/builder": "4.6.0",
+    "@skyux-sdk/builder-plugin-skyux": "4.1.4",
     "@skyux-sdk/e2e": "4.0.0",
-    "@skyux-sdk/testing": "4.2.0",
+    "@skyux-sdk/testing": "4.2.2",
     "@skyux/animations": "4.0.1",
     "@skyux/assets": "4.0.1",
-    "@skyux/config": "4.0.4",
-    "@skyux/core": "4.3.0",
-    "@skyux/docs-tools": "4.7.2",
-    "@skyux/forms": "4.13.2",
+    "@skyux/config": "4.1.0",
+    "@skyux/core": "4.3.1",
+    "@skyux/docs-tools": "4.7.4",
+    "@skyux/forms": "4.13.4",
     "@skyux/http": "4.0.1",
-    "@skyux/i18n": "4.0.2",
-    "@skyux/indicators": "4.5.0",
+    "@skyux/i18n": "4.0.3",
+    "@skyux/indicators": "4.7.0",
     "@skyux/inline-form": "4.1.0",
     "@skyux/layout": "4.3.0",
     "@skyux/lists": "4.3.0",
-    "@skyux/lookup": "4.5.0",
+    "@skyux/lookup": "4.8.0",
```

Comment:
We should update the peer dependency for `@skyux/lookup` to `^4.8.0` since the testing module requires the country fixture.